Streetcode/Streetcode.BLL/Services/Logging/LoggerService.cs

```diff
@@ -34,9 +34,16 @@ public void LogDebug(string msg)
 
         public void LogError(object request, string erroMsg)
         {
-            string requestType = request.GetType().ToString();
-            string requestClass = requestType.Substring(requestType.LastIndexOf('.') + 1);
-            _logger.Error($"{requestClass} handled with the error: {erroMsg}");
+            if (request != null)
+            {
+                string requestType = request.GetType().ToString();
+                string requestClass = requestType.Substring(requestType.LastIndexOf('.') + 1);
```

Comment:
1 is magic number, please use a var instead of it.

Reply:
It's just saying that substring should take everything after point, not including it. I think it looks as good as it can be here